integrationtests/server-integration/wildfly-modules/src/test/java/org/infinispan/test/integration/as/embedded/ReplicatedExampleIT.java

```diff
@@ -0,0 +1,60 @@
+package org.infinispan.test.integration.as.embedded;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+
+import org.infinispan.Cache;
+import org.infinispan.commons.util.Version;
+import org.infinispan.manager.DefaultCacheManager;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.infinispan.test.integration.data.Person;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.Asset;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.shrinkwrap.descriptor.api.Descriptors;
+import org.jboss.shrinkwrap.descriptor.api.spec.se.manifest.ManifestDescriptor;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(Arquillian.class)
+public class ReplicatedExampleIT {
+
+   @Deployment
+   public static Archive<?> deployment() {
+      return ShrinkWrap
+            .create(WebArchive.class, "simple.war")
+            .addClass(ReplicatedExampleIT.class)
+            .addClass(Person.class)
+            .addAsResource(
+                  ReplicatedExampleIT.class.getClassLoader().getResource("cache-config/replicated-example.xml"),
+                  "infinispan.xml")
```

Comment:
Can this be simplified to just `.addAsResource("cache-config/replicated-example.xml")`

Reply:
No. It will fail later
```
[ERROR] org.infinispan.test.integration.as.embedded.ReplicatedExampleIT.testPut  Time elapsed: 0.302 s  <<< ERROR!
java.io.FileNotFoundException: infinispan.xml (No such file or directory)
        at deployment.simple.war//org.infinispan.test.integration.as.embedded.ReplicatedExampleIT.testPut(ReplicatedExampleIT.java:46)
```

Reply:
Yes